2016/webcam/webcam.cbp

```diff
@@ -15,12 +15,12 @@
 					<Add option="-g" />
 				</Compiler>
 				<Linker>
-					<Add library="../../../../../usr/local/lib/libopencv_core.so" />
-					<Add library="../../../../../usr/local/lib/libopencv_highgui.so" />
-					<Add library="../../../../../usr/local/lib/libopencv_imgproc.so" />
-					<Add library="../../../../../usr/local/lib/libopencv_ts.a" />
-					<Add library="../../../../../usr/local/lib/libopencv_video.so" />
-					<Add library="../../../../../usr/local/lib/libopencv_videoio.so" />
+					<Add library="/home/benjamin/OpenCV Lib/libopencv_core.so" />
+					<Add library="/home/benjamin/OpenCV Lib/libopencv_highgui.so" />
+					<Add library="/home/benjamin/OpenCV Lib/libopencv_imgcodecs.so" />
+					<Add library="/home/benjamin/OpenCV Lib/libopencv_imgproc.so" />
+					<Add library="/home/benjamin/OpenCV Lib/libopencv_videoio.so" />
+					<Add library="/home/benjamin/2702_2016/2016/opencv_lib/bin/Debug/libopencv_lib.a" />
```

Comment:
We can't submit this, or else we'll have to fix it again once we're running on the laptop at school.  We should define an environment variable like %OPENCV_PATH%